.github/ISSUE_TEMPLATE/device-request.md

```diff
@@ -44,13 +44,34 @@ assignees: ''
   Required screenshots:
   - Endpoints and clusters of the node
   - Node Info panel
-  - Basic Cluster attributes in the Cluster Info panel.
 
   In the Cluster Info panel press "read" button to retreive the values. Please note that at least "Manufacturer Name" and "Model Identifier" must be populated with data (therefore, must not be empty), otherwise that information will not be usable. For battery powered devices, after pressing read it is required to wake-up the device by pressing a button or any other means of interaction.
 -->
 
 <!--
-  If available add screenshots of other clusters.
+  If available add screenshots of other clusters. Remove sections that are not available on your device.
 
   Relevant clusters are: Simple Metering, Electrical Measurement, Power Configuration, Thermostat, etc. You can typically spare Identify, Alarms, Device Temperature, On/Off. Please ensure data has been read prior to taking any screenshots.
 -->
+
+### Basic
+
+### Identify
+
+### Alarms
+
+### Device Temperature
+
+### Groups
+
+### Scenes
+
+### On/Off
+
+### Level Control
+
+### Color Control
+
+### Simple Metering
+
+### Diagnostics
```

Comment:
There are more i guess. So we need to either have the full list or add something like"Cluster 1, Cluster 2, Cluster 3, Etc."

Reply:
Right - I only have a relatively small number of device types. Are there docs elsewhere I can use to figure out this list? Otherwise, I can fill this in for the devices I do own.

Reply:
I think something like " Other clusters that are not mentioned above:"  would do. You named the most _common_ clusters already.